CorsixTH/Lua/map.lua

```diff
@@ -216,8 +216,12 @@ function Map:load(level, difficulty, level_name, map_file, level_intro, map_edit
         level_no = "0" .. level
       end
       -- Override with the specific configuration for this level
-      _, result = self:loadMapConfig(difficulty .. level_no .. ".SAM", base_config)
-      -- Finally load additional CorsixTH config per level
+      errors, result = self:loadMapConfig(difficulty .. level_no .. ".SAM", base_config)
+      if errors then
+        return nil, "Warning: Could not find campaign level " .. difficulty .. level_no ..
```

Comment:
No `.SAM` ?

A further improvement would be to use a separate variable for the filename, like you do at line 225 below. Doing that basically reduces the chance to create an inconsistency between what you load and what you report.